### PR TITLE
[TTAHUB-448] Specialist name filter

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

Add a 'Specialist name' filter to the AR landing and Regional Dashboard. We only add this filter if the user has the ability to approve reports.

## How to test

Create a user with the ability to approve reports. On both the AR Landing and Regional Dashboard create a filter using the new 'Specialist name' filter. Ensure relevant results are returned. 

Create a user who does NOT have the ability to approve reports. Both the AR Landing and Regional Dashboard should NOT have the ability to add the 'Specialist name' filter.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-448

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

**Review by GarrettEHill — APPROVED** [review]

**Comment by thewatermethod** [comment]
Do we want to bump .ttahub-filter-menu-item [name="condition"] to have a width of 150px instead of 140px?

(Screenshots show a width of 140, cutting off collaborator, and then a width of 150, showing it)

Screenshot 2023-07-11 at 10 24 13 AM

Screenshot 2023-07-11 at 10 25 05 AM

**Review by thewatermethod — CHANGES_REQUESTED** [review]
Seem to have found a bug. See attached video

https://github.com/HHS/Head-Start-TTADP/assets/3288586/255fbc76-2f33-4844-9edc-798244e49e23

File: src/scopes/activityReport/specialistName.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { Op } from 'sequelize';
+import { filterAssociation } from './utils';
```
Comment (thewatermethod) [inline src/scopes/activityReport/specialistName.js:2]: filterAssociation is unused

**Comment by thewatermethod** [comment]
Works more like I'd expect now - once that test is resolved, I'm good to approve

**Review by thewatermethod — APPROVED** [review]

**Pull request merged**